test/e2e/prometheus_client.go

```diff
@@ -212,15 +212,6 @@ func GetFirstValueFromPromQuery(body []byte) (float64, error) {
 		return 0, err
 	}
 
```

Comment:
The block comment doesn't match the body anymore, says "If body contains multiple timeseries
// GetFirstValueFromPromQuery errors."

Reply:
updated.